src/oauth-tooling.ts

```diff
@@ -44,7 +44,7 @@ const OAUTH_CONTENT_TYPE = 'application/x-www-form-urlencoded';
 function createAuthCodeRequestUri(authorizationEndpoint: string,
                                   redirectUri: string,
                                   clientId: string,
-                                  queryParams?: {}): string {
```

Comment:
Changelog relevant?

Reply:
Strictly speaking yes. Will add something there.

Reply:
LGTM beside the already mentioned points

Reply:
👍